### ISSUE TITLE
Links to download pdf and docx versions of guidance docs

### ISSUE THREAD

**Issue by LogiclyLinda** [logicly-au/docs.iar-dst.online#14]
@adamc00 I can't link to the pdf and docx files in a folder, it only seems to like https links (https://docs.iar-dst.online/en/v2/index.html).

for example, here's the link being used for another downloadable pdf - I don't know how to get there...

`https://docs.iar-dst.online/en/latest/_downloads/70063a1accf95b67ec00c864c41a1d09/National MH-IAR Guidance V1.05 - 30 July.pdf`

The files are attached here: https://rt.logicly.com.au/Transaction/Display.html?id=12843204

**Comment by adamc00** [comment]
Have you pushed a branch containing what you are trying to do @LindaTarr ?

**Comment by LogiclyLinda** [comment]
no I don't, it fails to build, so I just deleted it all and put in the index pages url to get it to build, bit dramatic

**Comment by adamc00** [comment]
Have a go at doing what you want in a branch and push that, doesn't matter that it fails. It'll be easier for me to understand what you are trying to do that way.

**Comment by LogiclyLinda** [comment]
@adamc00 here's my attempt: https://github.com/logicly-au/docs.iar-dst.online/tree/guidance-download-links

**Comment by adamc00** [comment]
That's sorted @LindaTarr 
* I hadn't realised that the `other-resources` directory had files that were to be published. I thought they were just for our reference. Anything that needs to be published has to be under the `docs` directory. I moved it there.
* There can't be a space after `:download:`, I've removed those
* For this kind of link you don't put an underscore `_` at the end. I've removed those.

See https://www.sphinx-doc.org/en/master/usage/referencing.html#role-download
